.github/workflows/osx_startup.yaml

```
@@ -0,0 +1,33 @@
+name: OSX
+on: [push, pull_request]
+jobs:
+
+  test:
+    name: Startup test
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: 'galaxy root'
+    - uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Cache pip dir
+      uses: actions/cache@v1
+      id: pip-cache
+      with:
+        path: ~/.cache/pip
```

```suggestion
        path: ~/Library/Caches/pip
```

That's where the pip cache is for macOS, see https://pip.pypa.io/en/latest/reference/pip_install/#caching . In fact, in "Post Cache pip dir" has this error:

```
/usr/bin/tar -cz -f /Users/runner/runners/2.165.2/work/_temp/11f2f806-b3e3-4d98-b798-325499fc12ea/cache.tgz -C /Users/runner/.cache/pip .
tar: could not chdir to '/Users/runner/.cache/pip'

[warning]Tar failed with error: The process '/usr/bin/tar' failed with exit code 1
```